FSExpExtension/MyVolume.swift

```diff
@@ -308,197 +285,168 @@ final class MyVolume: FSVolume, FSVolumeOperations {
     }
     
     private func mergeAttributes(_ existing: FSItemAttributes, request: FSItemSetAttributesRequest) {
-        if request.uidIsActive {
+        if request.wasConsumed(FSItemAttribute.UID) {
```

Comment:
These should use `isValid` instead of `wasConsumed` I believe?

Reply:
I'm not sure, to be honest. I still wasn't able to test this yet. Replaced with `isValid`.